tests/serialize/runstate/statemanager_test.py

```diff
@@ -83,6 +83,7 @@ def test_keys_for_items(self):
 
     def test_restore(self):
         job_names = ["one", "two"]
+        read_json = False
```

Comment:
we can probably move this to the setup function (https://github.com/Yelp/Tron/blob/df1dc3520e4fd63f880ac1c23e66b8529d24c627/tests/serialize/runstate/statemanager_test.py#L68) so that we don't need to set it for every test :)